packages/fx-core/templates/plugins/resource/aad/auth/V3/README.txt

```diff
@@ -459,21 +456,6 @@ Note: This part is for `command and response bot`.
       Note: Replace the value of "manifestTemplatePath" with the relative path of AAD app manifest noted in step 1.
             For example, './aad.manifest.template.json'
 
-  In `teamsfx/app.local.yml` only:
```

Comment:
why remove this part?

Reply:
Qianhao has a change in local debug last Friday and we will have bot domain instead of tunnel domain in .env.local file.